### PR TITLE
Update to-request-vpc-connectivity.adoc

### PR DESCRIPTION
**Pull request by sjsheridan**
Updating this based on customer feedback - current format was causing confusion. New format addresses those concerns. 
This also supersedes the change mentioned in #284

### CODE REVIEW

**Review by sjsheridan — COMMENTED** [review]
IPSec should be IPsec

Change: To update existing IPsec VPN connections that were created by MuleSoft Support to add new connections or update configurations:
To: To update existing IPsec VPN connections that were created by MuleSoft Support before November 2018:

Change all instances of: To avoid processing delays, ensure that you complete all required fields on both tabs.
Remove "required".

VPC Peering is possible when MuleSoft is the requestor.
This means the peering request must be established from the MuleSoft side. Customers should not create a peering request and ask us to accept it. 
Maybe better to say something like: VPC Peering requests should be established by MuleSoft. We do not accept peering requests.

Change: Support updates your ticket with the resulting peering ID when your request is complete.
To: Support updates your ticket to notify you when your request is complete.

This is not correct: To establish DX connections to your AWS account or partner’s AWS account and create a private Virtual Interface (VIF) request with the MuleSoft Account ID
What the customer needs to do is:
Establish Direct Connect connections to your own AWS account (or your partner's) AWS account
Create a private Virtual Interface (VIF) request towards the MuleSoft Account ID.
We cannot do anything until they have done that... then they fill out the form, etc. We can discuss this one via Slack if needed.

**Review by sjsheridan — COMMENTED** [review]
The headers should be just IPsec Connections, VPC Peering Connections, or Direct Connect Connections. The new or update part makes it too wordy, and that is called out in the points below.

Please remove this line: MuleSoft must all initiate VPC peering requests.
It’s not needed at all in retrospect.

Change: To request new or update existing VPC DX connections:
--> To request new or update existing DX connections:

**Comment by hannanelson** [comment]
> The headers should be just IPsec Connections, VPC Peering Connections, or Direct Connect Connections. The new or update part makes it too wordy, and that is called out in the points below.

Because these are tasks, we need to include the imperative. I tried to make them a bit shorter.

File: modules/ROOT/pages/to-request-vpc-connectivity.adoc [hunk 1]
```diff
@@ -3,39 +3,77 @@ ifndef::env-site,env-github[]
...
-[[request_vpc_connectivity]]
-== Request Connectivity for Anypoint VPCs
+== Update Connectivity for Existing IPsec VPN Connections
 
-To connect your Anypoint VPC to your internal network using VPN tunnel, peering, or direct connect, you must:
+As of November 2018, MuleSoft Support no longer creates new VPN connections.
+For all new VPNs, you must follow the self-service steps in xref:vpn-create-arm.adoc[Create an Anypoint VPN].
 
-. Download the link:https://docs.mulesoft.com/downloads/runtime-manager/VPC-form-v9.4-template.xlsx[Anypoint VPC discovery form], which requires at least MS Excel 2007 (with macros enabled). 
+To update *existing* IPsec VPN connections that were created by MuleSoft Support before November 2018:
```
Comment (kbostrom) [inline modules/ROOT/pages/to-request-vpc-connectivity.adoc]: Per style ref:
emphasis
Indicate emphasis using context alone; otherwise, use italics, but sparingly.
So, consider omitting or:
```suggestion
To update _existing_ IPsec VPN connections that were created by MuleSoft Support before November 2018:
```

File: modules/ROOT/pages/to-request-vpc-connectivity.adoc [hunk 2]
```diff
@@ -3,39 +3,77 @@ ifndef::env-site,env-github[]
...
+
+The MuleSoft Support team creates or updates the VPC peering connection request using the details provided in the VPC discovery form.
+Support updates your ticket to notify you when your request is complete.
+
+== Request Connectivity for Direct Connect Connections
+
+To request new or update existing Direct Connect (DX) connections:
+
+. Establish DX connections to your AWS account or your partner's AWS account.
+. Create a private Virtual Interface (VIF) request with the MuleSoft Account ID.
```
Comment (kbostrom) [inline modules/ROOT/pages/to-request-vpc-connectivity.adoc]:
```suggestion
. Create a private virtual interface (VIF) request with the MuleSoft Account ID.
```

File: modules/ROOT/pages/to-request-vpc-connectivity.adoc [hunk 3]
```diff
@@ -3,39 +3,77 @@ ifndef::env-site,env-github[]
...
+== Update Connectivity for Existing IPsec VPN Connections
 
-To connect your Anypoint VPC to your internal network using VPN tunnel, peering, or direct connect, you must:
+As of November 2018, MuleSoft Support no longer creates new VPN connections.
+For all new VPNs, you must follow the self-service steps in xref:vpn-create-arm.adoc[Create an Anypoint VPN].
 
-. Download the link:https://docs.mulesoft.com/downloads/runtime-manager/VPC-form-v9.4-template.xlsx[Anypoint VPC discovery form], which requires at least MS Excel 2007 (with macros enabled). 
+To update *existing* IPsec VPN connections that were created by MuleSoft Support before November 2018:
+
+. Download the link:https://docs.mulesoft.com/downloads/runtime-manager/VPC-form-v9.4-template.xlsx[Anypoint VPC discovery form].
```
Comment (kbostrom) [inline modules/ROOT/pages/to-request-vpc-connectivity.adoc:19]: This is the line that started all the trouble: Is the form for discovering ANY VPCs or only Anypoint VPCs?
Please search for "VPC" and evaluate whether your intention is clear. I tried, but I was not sure enough to commit....

File: modules/ROOT/pages/to-request-vpc-connectivity.adoc [hunk 4]
```diff
@@ -3,39 +3,77 @@ ifndef::env-site,env-github[]
...
++
+This form requires Microsoft Excel 2007 or later (with macros enabled).
+. For each VPC connection to update, complete and submit a separate VPC form:
```
Comment (kbostrom) [inline modules/ROOT/pages/to-request-vpc-connectivity.adoc]: Anypoint VPC connection?
Anypoint VPC form?
GLOBAL; I will stop asking.

**Review by kbostrom — APPROVED** [review]
Done. 
I think I actually deleted my change to your text, so no problem.
It was all the fault of VPC vs. Anypoint VPC!
